traits/traits.py

```diff
@@ -110,15 +110,17 @@ def password_editor ( auto_set=True, enter_set=False ):
 def multi_line_text_editor ( auto_set=True, enter_set=False ):
     """ Factory function that returns a text editor for multi-line strings.
     """
-    global MultilineTextEditor
+    global MultilineTextEditors
```

Comment:
We can rid of the this `global` statement now (we're only mutating the global `MultilineTextEditors` object, rather than rebinding the name).